src/utils/net.rs

```diff
@@ -44,19 +43,15 @@ pub async fn reader(location: &str) -> anyhow::Result<impl AsyncRead> {
             (Left(stream), content_length)
         }
         Err(_) => {
-            info!("reading file: {}", location);
+            info!("Reading file: {}", location);
             let stream = tokio::fs::File::open(location).await?;
             let content_length = stream.metadata().await?.len();
             (Right(stream), content_length)
         }
     };
 
-    if content_length > 0 {
-        read_progress.set_length(content_length);
-        read_progress.set_style(progress_style())
-    }
-
-    let mut reader = tokio::io::BufReader::new(read_progress.wrap_async_read(stream));
+    let mut reader =
+        tokio::io::BufReader::new(wrap_async_read("Reading ...", stream, content_length));
```

Comment:
We aren't just reading here. We're progressing a stream: Download/read -> uncompress -> write into DB. Maybe "Loading" would be a better description.